.github/workflows/tests.yml

```diff
@@ -356,7 +366,7 @@ jobs:
 
       - run: |
           set -o pipefail
-          COMPLEMENT_DIR=`pwd`/complement synapse/scripts-dev/complement.sh -json 2>&1 | gotestfmt
+          WORKERS=${{ matrix.workers && 1 }} COMPLEMENT_DIR=`pwd`/complement synapse/scripts-dev/complement.sh -json 2>&1 | gotestfmt
```

Comment:
might be worth sticking `-p 1` on the commandline here, at least for the workers case, to see if it makes things go faster (cf https://github.com/matrix-org/synapse/issues/12638#issuecomment-1128773155)

Reply:
Time without `-p 1`
* Monolith: 8m 59s
* Workers: 52m 2s

Time with `-p 1`: TBC

Reply:
Surprisingly it seemed to make things worse (unless it's just that noisy).

Reply:
eh, not that surprising. It was just a straw to grasp at.